extensions/resource-deployment/src/ui/deployClusterWizard/pages/clusterSettingsPage.ts

```diff
@@ -39,19 +39,20 @@ export class ClusterSettingsPage extends WizardPageBase<DeployClusterWizard> {
 					useCustomValidator: true
 				}, {
 					type: FieldType.Text,
-					label: localize('deployCluster.ControllerUsername', "Controller username"),
+					label: localize('deployCluster.AdminUsername', "Admin username"),
 					required: true,
 					variableName: VariableNames.AdminUserName_VariableName,
 					defaultValue: 'admin',
-					useCustomValidator: true
+					useCustomValidator: true,
+					description: localize('deployCluster.AdminUsernameDescription', "You can use this username to access controller and SQL Server, username for gateway is root.")
 				}, {
```

Comment:
I'd run this by PMs, but I think it reads better as "You can use this username to access the controller..."

Reply:
@dnethi what do you think?

Reply:
I agre with Charles that putting a "the" makes it seem nicer.

In addition, I'd change the "," after "Server" to a ".", and capitalize the "U" in "Username"